### DISCUSSION TITLE
Bluetooth BT_LE_ADV_OPT_EXT_ADV option use.

### DISCUSSION THREAD

**Discussion by 2songh** [zephyrproject-rtos/zephyr#55809] [Q&A]
**Is your feature request related to a problem? Please describe.**
I wish use BT_LE_ADV_OPT_EXT_ADV option.

**Environment**
Board: nucleo_wb55rg
OS: Linux ubuntu 18.04
Versioin: Zephyr 2.4.0

**Describe the solution you'd like.**
I want advertise with extended advertising.

**prj.conf**
![image](https://user-images.githubusercontent.com/126455296/225256041-a41e0816-3c7f-46f1-84a8-6c188f12cdf0.png)

**explanation**
![image](https://user-images.githubusercontent.com/126455296/225252855-9c88a91c-5512-4a52-9d23-0af36a211d67.png)
*****
![image](https://user-images.githubusercontent.com/126455296/225253426-9dc60f9e-d29e-49eb-baa4-ed37aa10acdd.png)
*****
**bt_le_adv_start(param, ad, ARRAY_SIZE(ad), NULL, 0);**

And I get error.
**Advertising failed to start (err -22)**

How can I solve this error?

**Comment by hermabe** [comment]
This is not a feature request, but a question. Moving to discussion.

`(err -22)` is `-EINVAL`, so the parameters you have given are somehow invalid. Is extended advertising supported and enabled in the controller you use?
